mmv1/third_party/terraform/services/dns/data_source_dns_managed_zone.go

```diff
@@ -165,7 +166,7 @@ func (d *GoogleDnsManagedZoneDataSource) Read(ctx context.Context, req datasourc
 		}
 	}
 
-	tflog.Trace(ctx, "read dns record set data source")
+	tflog.Trace(ctx, "read dns managed zone data source")
```

Comment:
A copy-paste error I'm fixing in this PR - doesn't affect the data source's function so I'm not going to list it in the release notes.